network.js

```diff
@@ -43,16 +43,15 @@ function setJavaProxyArgs(fields) {
     `-Dhttp.proxyHost=${fields.hostname}`,
     `-Dhttp.proxyPort=${fields.port}`,
     `-Dhttps.proxyHost=${fields.hostname}`,
-    `-Dhttps.proxyPort=${fields.port}`
+    `-Dhttps.proxyPort=${fields.port}`,
+    `-Dhttp.nonProxyHosts="localhost|127.0.0.1|0.0.0.0"`
```

Comment:
@tejohnso but that doesn't include addresses in local networks (for example 192.168.1.1). I guess the question is should we exclude those addresses or not. We are excluding them in the pac file, but not here.